pages/blueprint/index.js

```diff
@@ -446,6 +453,7 @@ const makeMapStateToProps = () => {
       componentsSortKey: state.sort.components.key,
       componentsSortValue: state.sort.components.value,
       componentsFilters: state.filter.components,
+      blueprintContentsError: {},
```

Comment:
@jkozol I wasn't sure what I should put here. The initial state is null for `blueprints.errorState` and if it's anything but null, it will display an error message where the blueprint contents should display. I think setting this property to `{}` is the right behavior for this case, but would like your input here.

Reply:
That seems like it should work for an empty error state. Are you opposed to initializing the errorState to an empty object as well? If we have the default being an empty object I feel like the initial state should be an empty object too. I'm not really sure what the standard should be though so I'm fine either way.

Reply:
Can you clarify what's happening for this set of MapStateToProps?

Currently, the error state displays if anything except `null` is defined. I'm making an assumption that if `if (getBlueprintById(state, props.route.params.blueprint.replace(/\s/g, '-')) !== undefined)` does resolve to undefined, then triggering the error state would be the right thing to do, hence setting it to `{}`, but I could be misinterpreting something here.

For the actual api call, if an error is returned, do you know if we get something back besides just `{}`? Is there a reliable way to test error handling?

Reply:
Okay. That makes sense to me. I haven't tested for error handling yet so I'm unsure.